presidio-anonymizer/presidio_anonymizer/entities/engine_request.py

```diff
@@ -0,0 +1,98 @@
+"""
+Engine request entity.
+
+It get the data and validate it before the engine receives it.
+"""
+import logging
+
+from presidio_anonymizer.anonymizers.fpe import FPE
+from presidio_anonymizer.anonymizers.hash import Hash
+from presidio_anonymizer.anonymizers.mask import Mask
+from presidio_anonymizer.anonymizers.redact import Redact
+from presidio_anonymizer.anonymizers.replace import Replace
+from presidio_anonymizer.entities.analyzer_result import AnalyzerResult
+from presidio_anonymizer.entities.analyzer_results import AnalyzerResults
+from presidio_anonymizer.entities.invalid_exception import InvalidParamException
+
+
+class AnonymizerEngineRequest:
+    """The request the engine will get and act on."""
+
+    anonymizers = {"mask": Mask, "fpe": FPE, "replace": Replace, "hash": Hash,
+                   "redact": Redact}
+
+    logger = logging.getLogger("presidio-anonymizer")
+
+    def __init__(self,
+                 data: dict):
+        """Handle text replacement for PIIs with requested transformations.
```

Comment:
This is a constructor, so I think this doc is more relevant to the class level doc.
And I also think that it's not "replacement" only.
I would just remove this line.

Reply:
I can't remove this line :( linter...